### PR TITLE
Update monero_it.ts

### PR DESCRIPTION
**Pull request by hrumag**
Aggiunte traduzioni dove "unfinished".
Correzione traduzioni.

Added translation where "unfinished".
Corrected some translations.

### CODE REVIEW

**Comment by cryptochangements34** [comment]
Ping @erciccione and @serhack

**Comment by serhack** [comment]
Thanks @cryptochangements! 
Grazie @hrumag

File: translations/monero_it.ts [hunk 1]
```diff
@@ -155,23 +155,23 @@
         <location filename="../src/wallet/api/wallet.cpp" line="1197"/>
         <location filename="../src/wallet/api/wallet.cpp" line="1301"/>
         <source>not enough outputs for specified ring size</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">insufficiente numero di output per il ring size specificato</translation>
```
Comment (erciccione) [inline translations/monero_it.ts]: Please remove `type="unfinished"`

File: translations/monero_it.ts [hunk 2]
```diff
@@ -155,23 +155,23 @@
...
     </message>
     <message>
         <location filename="../src/wallet/api/wallet.cpp" line="1199"/>
         <location filename="../src/wallet/api/wallet.cpp" line="1303"/>
         <source>found outputs to use</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">trovati output che possono essere usati</translation>
```
Comment (erciccione) [inline translations/monero_it.ts]: please remove `type="unfinished"`

File: translations/monero_it.ts [hunk 3]
```diff
@@ -462,7 +462,7 @@
     <message>
         <location filename="../src/rpc/rpc_args.cpp" line="105"/>
         <source> requires RFC server password --</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished"> richiede la password del server RFC --</translation>
```
Comment (erciccione) [inline translations/monero_it.ts]: Please remove `type="unfinished"`

File: translations/monero_it.ts [hunk 4]
```diff
@@ -118,32 +118,32 @@
...
         <location filename="../src/wallet/api/wallet.cpp" line="1111"/>
         <source>payment id has invalid format, expected 16 or 64 character hex string: </source>
-        <translation>L&apos;id pagamento è in un formato invalido, dovrebbe essere una stringa hex di 16 o 64 caratteri: </translation>
+        <translation>L&apos;ID pagamento è in un formato invalido, dovrebbe essere una stringa esadecimale di 16 o 64 caratteri: </translation>
     </message>
     <message>
         <location filename="../src/wallet/api/wallet.cpp" line="1121"/>
         <source>Failed to add short payment id: </source>
-        <translation>Impossibile aggiungere id pagamento corto</translation>
+        <translation>Impossibile aggiungere ID pagamento corto</translation>
```
Comment (erciccione) [inline translations/monero_it.ts]: Please add `:`

File: translations/monero_it.ts [hunk 5]
```diff
@@ -819,37 +819,37 @@ Questa transazione verrà sbloccata al blocco %llu, in approssimativamente %s gi
     <message>
         <location filename="../src/simplewallet/simplewallet.cpp" line="4763"/>
         <source>Loaded %lu transactions, for %s, fee %s, %s, %s, with min ring size %lu, %s. %sIs this okay? (Y/Yes/N/No): </source>
-        <translation type="unfinished"></translation>
+        <translation>Caricate %lu transazioni, per %s, commissione %s, %s, %s, con ring size minimo %lu, %s. %sOK?(Y/Yes/N/No)</translation>
```
Comment (erciccione) [inline translations/monero_it.ts]: Please add `:`

File: translations/monero_it.ts [hunk 6]
```diff
@@ -39,17 +39,17 @@
     <message>
         <location filename="../src/wallet/api/pending_transaction.cpp" line="115"/>
         <source>daemon is busy. Please try again later.</source>
-        <translation>Il daemon è impegnato. Prova più tardi.</translation>
+        <translation>il demone è impegnato. Prova più tardi.</translation>
```
Comment (erciccione) [inline translations/monero_it.ts]: Please keep `daemon`, i's left in the original form also in the GUI

File: translations/monero_it.ts [hunk 7]
```diff
@@ -39,17 +39,17 @@
...
     </message>
     <message>
         <location filename="../src/wallet/api/pending_transaction.cpp" line="118"/>
         <source>no connection to daemon. Please make sure daemon is running.</source>
-        <translation>Nessuna connessione con il daemon. Controlla che sia operativo.</translation>
+        <translation>nessuna connessione con il demone. Controlla che sia operativo.</translation>
```
Comment (erciccione) [inline translations/monero_it.ts]: daemon

File: translations/monero_it.ts [hunk 8]
```diff
@@ -39,17 +39,17 @@
...
     </message>
     <message>
         <location filename="../src/wallet/api/pending_transaction.cpp" line="122"/>
         <source>transaction %s was rejected by daemon with status: </source>
-        <translation>La transazione %s è stata respinta dal daemon con status: </translation>
+        <translation>la transazione %s è stata respinta dal demone con status: </translation>
```
Comment (erciccione) [inline translations/monero_it.ts]: daemon

File: translations/monero_it.ts [hunk 9]
```diff
@@ -118,32 +118,32 @@
...
         <source>Failed to add short payment id: </source>
-        <translation>Impossibile aggiungere id pagamento corto</translation>
+        <translation>Impossibile aggiungere ID pagamento corto</translation>
     </message>
     <message>
         <location filename="../src/wallet/api/wallet.cpp" line="1154"/>
         <location filename="../src/wallet/api/wallet.cpp" line="1258"/>
         <source>daemon is busy. Please try again later.</source>
-        <translation>il daemon è impegnato. Prova più tardi</translation>
+        <translation>il demone è impegnato. Riprova più tardi.</translation>
```
Comment (erciccione) [inline translations/monero_it.ts]: daemon

File: translations/monero_it.ts [hunk 10]
```diff
@@ -118,32 +118,32 @@
...
     </message>
     <message>
         <location filename="../src/wallet/api/wallet.cpp" line="1157"/>
         <location filename="../src/wallet/api/wallet.cpp" line="1261"/>
         <source>no connection to daemon. Please make sure daemon is running.</source>
-        <translation>nessuna connessione con il daemon. Accertati che sia operativo</translation>
+        <translation>nessuna connessione con il demone. Accertati che sia operativo.</translation>
```
Comment (erciccione) [inline translations/monero_it.ts]: daemon

File: translations/monero_it.ts [hunk 11]
```diff
@@ -252,39 +252,39 @@
     <message>
         <location filename="../src/wallet/api/wallet.cpp" line="986"/>
         <source>Key images can only be imported with a trusted daemon</source>
-        <translation type="unfinished"></translation>
+        <translation>Le key image possono essere importate solo con un demone fidato</translation>
```
Comment (erciccione) [inline translations/monero_it.ts]: daemon

File: translations/monero_it.ts [hunk 12]
```diff
@@ -551,12 +551,12 @@
     <message>
         <location filename="../src/simplewallet/simplewallet.cpp" line="2635"/>
         <source>wallet failed to connect to daemon: </source>
-        <translation>impossibile connettere il portafoglio al daemon: </translation>
+        <translation>impossibile connettere il portafoglio al demone: </translation>
```
Comment (erciccione) [inline translations/monero_it.ts]: daemon

File: translations/monero_it.ts [hunk 13]
```diff
@@ -551,12 +551,12 @@
...
     </message>
     <message>
         <location filename="../src/simplewallet/simplewallet.cpp" line="2643"/>
         <source>Daemon uses a different RPC major version (%u) than the wallet (%u): %s. Either update one of them, or use --allow-mismatched-daemon-version.</source>
-        <translation>Il daemon usa una versione principale RPC (%u) diversa da quella del portafoglio (%u): %s. Aggiorna una delle due, o usa --allow-mismatched-daemon-version.</translation>
+        <translation>Il demone usa una versione principale RPC (%u) diversa da quella del portafoglio (%u): %s. Aggiorna una delle due, o usa --allow-mismatched-daemon-version.</translation>
```
Comment (erciccione) [inline translations/monero_it.ts]: daemon

File: translations/monero_it.ts [hunk 14]
```diff
@@ -1202,59 +1202,59 @@ Questa transazione verrà sbloccata al blocco %llu, in approssimativamente %s gi
...
         <location filename="../src/simplewallet/simplewallet.cpp" line="980"/>
         <source>Failed to update spent status after importing multisig info: </source>
-        <translation type="unfinished"></translation>
+        <translation>Impossibile aggiornare lo stato di spesa dopo aver importato le informazioni sul multisig: </translation>
     </message>
     <message>
         <location filename="../src/simplewallet/simplewallet.cpp" line="985"/>
         <source>Untrusted daemon, spent status may be incorrect. Use a trusted daemon and run &quot;rescan_spent&quot;</source>
-        <translation type="unfinished"></translation>
+        <translation>Demone non fidato, lo stato di spesa potrebbe non essere corretto. Usare un demone fidato ed eseguire &quot;rescan_spent&quot; </translation>
```
Comment (erciccione) [inline translations/monero_it.ts]: Daemon

**Comment by hrumag** [comment]
Ciao @erciccione, I fixed the points you asked me to. I went further in translating other "unfinished" lines.

File: translations/monero_it.ts [hunk 15]
```diff
@@ -1365,22 +1365,22 @@ Questa transazione verrà sbloccata al blocco %llu, in approssimativamente %s gi
...
         <location filename="../src/simplewallet/simplewallet.cpp" line="1586"/>
         <source>Show the incoming transfers, all or filtered by availability and address index.</source>
-        <translation type="unfinished"></translation>
+        <translation>Mostra i trasferimenti in entrata, tutti o filtrati per disponibilità ed indice di indirizzo.</translation>
     </message>
     <message>
         <location filename="../src/simplewallet/simplewallet.cpp" line="1589"/>
         <source>payments &lt;PID_1&gt; [&lt;PID_2&gt; ... &lt;PID_N&gt;]</source>
-        <translation type="unfinished"></translation>
+        <translation type=>pagamenti &lt;PID_1&gt; [&lt;PID_2&gt; ... &lt;PID_N&gt;]</translation>
```
Comment (erciccione) [inline translations/monero_it.ts]: you forgot `type=`

**Comment by NanoAkron** [comment]
Squash commits?

**Pull request merged**